README.md

```diff
@@ -1,4 +1,34 @@
 # 🔥_Fireselect_
+Vue component to `<select>` box.  
```

Comment:
I'd use something more in the line of:
Vue component that transforms overwhelming `<select>` boxes into something fancy, simple and user-friendly.